plugins/Eq/EqEffect.cpp

```diff
@@ -313,41 +312,41 @@ float EqEffect::peakBand( float minF, float maxF, EqAnalyser *fft, int sr )
 void EqEffect::setBandPeaks( EqAnalyser *fft, int samplerate )
 {
 	m_eqControls.m_lowShelfPeakR = m_eqControls.m_lowShelfPeakL =
-			peakBand( m_eqControls.m_lowShelfFreqModel.value()
+			linearPeakBand( m_eqControls.m_lowShelfFreqModel.value()
```

Comment:
```suggestion
			linearPeakBand(m_eqControls.m_lowShelfFreqModel.value()
```
Along with the others below

Reply:
Fixed with commit f2d63ecca1b which also introduced a lambda to reduce code repetition.